app/javascript/react/containers/BoardContainer.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react"
+import CategoryTile from "../components/CategoryTile"
+
+const BoardContainer = (props) => {
+  const[categories, setCategories] = useState([])
+  // const[currentUserId, setCurrentUserId] = useState(null)
+
+  useEffect(()=> {
+    fetch("/api/v1/categories")
+    .then((response)=> {
+      if (response.ok) {
+        return response
+      } else {
+        let errorMessage = `${response.status} (${response.statusText})`,
+        error = new Error(errorMessage)
+        throw(error)
+      }
+    })
+    .then(response => response.json())
+    .then(Body => {
+      setCategories(Body.categories)
+    })
+    .catch(error => console.error(`Error in fetch: ${error.message}`))
+  }, [])
```

Comment:
when actioncable is in, I would update your controller so that in the response we also get the current user